lib/iris/tests/unit/coord_categorisation/test_add_categorised_coord.py

```diff
@@ -116,6 +116,11 @@ def make_cube(self, calendar):
 
     def test_calendars(self):
         for calendar in calendars:
+            # Skip the Julian calendar due to
+            # https://github.com/Unidata/netcdftime/issues/13
+            # Remove this if block once the issue is resolved.
```

Comment:
A parallel in our own project is where we pin the numpy version to v1.11 until an iris issue is resolved:
https://github.com/SciTools/iris/issues/2425